module03/solution.py

```diff
@@ -0,0 +1,96 @@
+"""Solutions of module 3"""
+import math
+
+def sum_of_digits(num):
+    """Return sum of digits of an integer n."""
+    num = abs(num)
+    summ = 0
+    while num > 0:
+        summ += num%10
+        num //= 10
+    return summ
+
+def reverse_list(lst_orig):
+    """Return list in reverse order"""
+    lst_rev = []
+    while lst_orig:
+        lst_rev.append(lst_orig.pop())
+    return lst_rev
+
+def to_digits(num):
+    """Return a list of digits of an integer n"""
+    num = abs(num)
+    digits = []
+    while num > 0:
+        digits.append(num%10)
+        num //= 10
+    return reverse_list(digits)
+
+def to_number(digits):
+    """Return a number based on digits"""
+    digits = reverse_list(digits)
+    num = 0
+    for i, val in enumerate(digits):
+        num += val * 10 ** i
+    return num
+
+def count_vowels(strng):
+    """Return count ov vovels in a string"""
+    vowels = 'aeiouy'
+    num_vowels = 0
+    for char in strng.lower():
+        if char in vowels:
+            num_vowels += 1
+    return num_vowels
+
+def count_consonants(strng):
+    """Return count ov consonants in a string"""
+    consonants = 'bcdfghjklmnpqrstvwxz'
+    num_consonants = 0
+    for char in strng.lower():
+        if char in consonants:
+            num_consonants += 1
+    return num_consonants
+
+def prime_number(num):
+    """Return true if n is prime number"""
+    if num == 2:
+        return True
+    for i in range(2, num):
+        if num%i == 0:
+            return False
+    return True
+
+def fact_digits(num):
+    """Return the sum of factoriel of each digit of a numner"""
+    sum_factorial = 0
+    for digit in to_digits(num):
+        sum_factorial += math.factorial(digit)
+    return sum_factorial
+
+def fibonacci(num):
+    """Return num-th member of the Fobonacci sequence"""
+    fibonacci_seq = [1, 1]
+    for i in range(2, num + 1):
+        fibonacci_seq.append(fibonacci_seq[-1] + fibonacci_seq[-2])
+    return fibonacci_seq[0:num]
+
+def fib_number(num):
+    """Return a string of Fibonacci sequence to num-th member"""
+    fibonacci_str = "".join(map(str, fibonacci(num)))
+    return int(fibonacci_str)
```

Comment:
When the task states *Return a string ...* why do you run **int()** over it?

Reply:
I misread the task, an integer is required. I'll update the comment.